dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jaxwscdi-2.3.feature

```diff
@@ -8,6 +8,6 @@ IBM-Provision-Capability: \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.cdi-2.0))"
 IBM-Install-Policy: when-satisfied
```

Comment:
This auto feature goes away now doesn't it?

Reply:
Or actually the update to com.ibm.websphere.appserver.jaxwscdi-2.2.feature  should be undone I guess.

Reply:
You did not do the change to com.ibm.websphere.appserver.jaxwscdi-2.2.feature